src/test/java/teammates/test/cases/storage/InstructorsDbTest.java

```diff
@@ -318,8 +319,10 @@ public void testUpdateInstructorByGoogleId() throws Exception {
             signalFailureToDetectException();
         } catch (InvalidParametersException e) {
             AssertHelper.assertContains(
-                        String.format(FieldValidator.PERSON_NAME_ERROR_MESSAGE, instructorToEdit.name, FieldValidator.REASON_EMPTY) + Const.EOL
-                        + String.format(FieldValidator.EMAIL_ERROR_MESSAGE, instructorToEdit.email, FieldValidator.REASON_INCORRECT_FORMAT),
+                        String.format(FieldValidator.PERSON_NAME_ERROR_MESSAGE, instructorToEdit.name,
+                                      FieldValidator.REASON_EMPTY) + Const.EOL
+                        + String.format(FieldValidator.EMAIL_ERROR_MESSAGE, instructorToEdit.email,
+                                        FieldValidator.REASON_INCORRECT_FORMAT),
```

Comment:
Maybe indent these 4 spaces to the right, as they aren't on the same level as `assertContains`